docs/discover/search.asciidoc

```diff
@@ -88,6 +88,67 @@ If the saved search is associated with a different index pattern than is current
 selected, opening the saved search changes the selected index pattern. The query language
 used for the saved search will also be automatically selected.
 
+[[save-load-delete-query]]
+=== Saving and Loading Queries
+Saving queries enables you to reload them into <<discover, Discover>>, <<visualize, visualizations>> and <<dashboard, dashboards>>.
```

Comment:
I'd suggest a slight tweak: 

```suggestion
Saving queries allows you to reuse them in <<discover, Discover>>, <<visualize, Visualize >> and <<dashboard, Dashboard>>.
```

Using the terms `visualizations` and `dashboards` made me think they could somehow be included in the individual dashboard and visualization objects themselves.

Reply:
https://github.com/elastic/kibana/pull/43731/commits/b3163d0499ebb516e91879a34d4508e9e5deca57

Reply:
I'm still seeing `visualizations and dashboards` instead of `Visualize and Dashboard`

Reply:
https://github.com/elastic/kibana/pull/43731/commits/1924aaebb70e917ca6dc5682ea2af9fcb8391d85#diff-87a4343c7de08863ed7fa9d869b5c5f1R91